tools/bwa_index.cwl

```diff
@@ -0,0 +1,56 @@
+class: CommandLineTool
+cwlVersion: v1.0
+id: bwa_mem_naive
```

Comment:
I might have missed this elsewhere, but this tool name is misleading

Reply:
Double checked and it looks like this is the only one I missed.